README.md

```diff
@@ -1268,6 +1268,14 @@ aView.translateSubviews()
 It will iterate on all the subviews of the view, and use the text / placeholder as key in `NSLocalizedString`.
 By settings your localizable key in your xib / storyboard, all yours string will be automatically translated just by calling the above method.
 
+Add constraints between a view and its superview:
+
+```swift
+aView.addConstraints() // Add constraints to all edges with zero insets
+aView.addConstraints(to: [.top, .bottom]) // Add constraints to top and bottom edges with zero insets
+aView.addConstraints(to: [.top, .bottom], insets: .zero) // Add constraints to top and bottom edges with zero insets
```

Comment:
Maybe pass a `UIEdgeInset` with some non-zero values in the last example? Otherwise it's kind of the same as the second example...

Reply:
Good idea, will do